gcn_monitor/kafka.py

```diff
@@ -44,3 +53,5 @@ def run():
                 log.info("topic %s: got message", topic)
                 partition = message.partition()
                 metrics.received.labels(topic, partition).inc()
+                key = f"{topic}-{datetime.now(timezone.utc)}"
+                s3_client.put_object(Bucket=BUCKET_NAME, Key=key, Body=message.value())
```

Comment:
We also need to capture the message key, timestamp, partition, and any metadata.